src/helpers/buildHyperlinkHelper.ts.test.ts

```diff
@@ -1,107 +1,93 @@
 import { buildHyperlink } from "./buildHyperlinkHelper";
 import type { SelectedGeographyData } from "../types";
 
-const mockSelectedGeography = {
-  geoType: "lad",
-  displayName: "Lad name",
-  geoCode: "ladCode",
-  variableData: {
-    catCode: { count: 0, percentage: 0, total: 0 },
-  },
-} as SelectedGeographyData;
+const ladSelectedUrl = new URL("/test?lad=ladCode", "https://test.com");
+const ewSelectedUrl = new URL("/test", "https://test.com");
 
 test("returns correct url given all inputs", () => {
-  const args = {
+  const urlParams = {
     topic: "topic",
     variable: "variable",
     classification: "classification",
     category: "category",
-    selectedGeography: mockSelectedGeography,
   };
-  expect(buildHyperlink(args)).toEqual("/2021/topic/variable/classification/category?lad=ladCode");
+  expect(buildHyperlink(ladSelectedUrl, urlParams)).toEqual("/2021/topic/variable/classification/category?lad=ladCode");
 });
 
 describe("index page url", () => {
-  test("returns base url given no inputs", () => {
-    expect(buildHyperlink({})).toEqual("/");
-  });
-  test("returns base url with an LAD selectedGeography but no other arguments", () => {
-    expect(buildHyperlink({ selectedGeography: mockSelectedGeography })).toEqual("/");
+  test("returns base url given only url input", () => {
+    expect(buildHyperlink(ladSelectedUrl)).toEqual("/?lad=ladCode");
   });
 });
 
 describe("topic url", () => {
-  let args = {
+  let urlParams = {
```

Comment:
linters telling me these should be constants (optional obvs if you disagree)

Reply:
this is a hangover from when I was modifying the args between tests - will change